staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go

```diff
@@ -945,6 +967,20 @@ func (alloc *allocator) allocateDevice(r deviceIndices, device deviceWithID, mus
 		return false, nil, nil
 	}
 
+	// the API validation logic has in place to check the ConsumesCounter referred should exist inside SharedCounters
```

Comment:
Nit: if you write full sentences, capitalize and end with a full stop. Also, consider breaking up long lines where it makes sense.

Reply:
Also "has in place to check"?

Reply:
Update comment.